src/CarbonAware/src/Model/Location.cs

```diff
@@ -31,21 +26,6 @@ public class Location
     public string? RegionName { get; set; }
```

Comment:
For clarity, can we use the term `Name` here rather than `RegionName` as this is no longer supporting just cloud providers?

Reply:
hmm, that means we need to change the source file as well, so the deserialization works, and also by just changing the name, it changed another bunch of files. Do you still want this to change?

Reply:
It is a large find/replace, but I think ultimately will make this codebase clearer and less prescriptive in line with this particular feature.

I thought deserialization is tied to the `NamedGeoposition` model.  Is that not true? That model could change too, but I am more interested in the clarity of the core `Location` model.

Reply:
> It is a large find/replace, but I think ultimately will make this codebase clearer and less prescriptive in line with this particular feature.
> 
> I thought deserialization is tied to the `NamedGeoposition` model. Is that not true? That model could change too, but I am more interested in the clarity of the core `Location` model.

Oh, right. `NamedGeoposition` is the one doing the deserialization. I'll change then the `Location`. Thx